src/openapiGenerator/templates/typescript-angular/api.service.mustache

```diff
@@ -0,0 +1,546 @@
+{{>licenseInfo}}
+/* tslint:disable:no-unused-variable member-ordering */
+
+import { Inject, Injectable, Optional }                      from '@angular/core';
+{{#useHttpClient}}
+import { HttpClient, HttpHeaders, HttpParams,
+         HttpResponse, HttpEvent, HttpParameterCodec }       from '@angular/common/http';
+import { CustomHttpParameterCodec }                          from '../encoder';
```

Comment:
The imports here looks like project specific , or is it something common to all angular projects ? If its common to angular projects, at the very least can we typescript us modular importation instead like 
`import { CustomHttpParameterCodec }                          from '@utils/encoder';`

Reply:
that part is unchanged from the openapi mustache file https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/typescript-angular/api.service.mustache#L7

It points to this file which also gets generated and is reused in all api services 
https://github.com/OpenAPITools/openapi-generator/blob/ec39005ae528f88522caedd91db91c0b59624d94/samples/client/petstore/typescript-angular-v6-not-provided-in-root/builds/default/encoder.ts